modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -462,4 +463,12 @@ export abstract class BaseCoin implements IBaseCoin {
   getInscriptionBuilder(wallet: Wallet): IInscriptionBuilder {
     throw new NotImplementedError('Inscription Builder is not supported for this coin');
   }
+
+  /**
+   * Function to get coin specific hash function used to generate transaction digests.
+   * @returns {@see Hash} hash function if implemented, otherwise undefined
+   */
+  getHashFunction(): Hash | undefined {
+    return undefined;
```

Comment:
```suggestion
    throw new NotImplementedError('getHashFunction is not supported for this coin');
```

Reply:
With this suggestion we can also change the return type to be just `Hash`.

Reply:
done